examples/app.py

```diff
@@ -49,17 +65,27 @@
 from __future__ import absolute_import, print_function
 
 from flask import Flask, render_template
+
 from flask_babelex import Babel
+
 from flask_cli import FlaskCLI
+
 from flask_mail import Mail
+
 from flask_security import current_user
-from invenio_db import InvenioDB
 
 from invenio_accounts import InvenioAccounts
 
+from invenio_assets import InvenioAssets
+
+from invenio_db import InvenioDB
```

Comment:
Can you wrap the imports in a:

```
try:
except ImportError:
```

so that the example app also works without Invenio Theme/Assets installed.